engine-rest/engine-rest-openapi/src/main/templates/paths/history/identity-link-log/get.ftl

```diff
@@ -0,0 +1,85 @@
+<#-- Generated From File: camunda-docs-manual/public/reference/rest/history/identity-links/get-identity-link-query/index.html -->
+<#macro endpoint_macro docsUrl="">
+{
+  <@lib.endpointInfo
+      id = "getHistoricIdentityLinks"
+      tag = "Historic Identity Link Log"
+      summary = "Get Identity Link Logs"
+      desc = "Queries for historic identity link logs that fulfill given parameters.
+              The size of the result set can be retrieved by using the
+              [Get Identity-Link-Log Count](${docsUrl}/reference/rest/history/identity-links/get-identity-link-query-count/)
+              method."
+  />
+
+  "parameters" : [
+
+    <#assign last = false >
+    <#include "/lib/commons/history-identity-link.ftl" >
+    <@lib.parameters
+        object = params
+        last = last
+    />
+    <#assign last = true >
+    <#include "/lib/commons/sort-params.ftl">
```

Comment:
Pagination parameters are missing. They are also missing in the docs

Reply:
Good catch, thanks a lot! I will add them 👍